unittests/CMakeLists.txt

```diff
@@ -47,7 +47,6 @@ include (add_pstore)
 
 add_library (pstore-unit-test-harness STATIC harness.cpp)
 set_target_properties (pstore-unit-test-harness PROPERTIES FOLDER "pstore test libraries")
-set_property (TARGET pstore-unit-test-harness PROPERTY CXX_STANDARD 11)
```

Comment:
I think that we should explicitly request C++14 rather than accept the compiler default.